### DISCUSSION TITLE
[Pathway Discussion]: WP5424 - How to represent metabolites that are excreted into the urine?

### DISCUSSION THREAD

**Discussion by JuliaBatUM** [wikipathways/wikipathways-help#82] [pathway-discussion]
### Discussion

Hi!
My project goal is to represent oxylipins and the link to senescence on a map. During literature research, I found many metabolites to be secreted into the urine and thought it would also be interesting to represent that on the map (it would also be useful for us later to add it to human1).
For simplicity, I used an arrow (basic interactions) pointing to a pink pathway called "Excretion via Urine" :
![image](https://github.com/wikipathways/wikipathways-help/assets/130973169/c30d34e2-7064-45cb-9ac0-9e796e36cc3f)

Is there a better way to represent metabolites that are excreted?

During the meeting (02/11/23), we discussed to:

- add a secretion reaction and have the metabolite drawn outside the cell -> I think this would make the map even more unclear as it would add more reactions and more arrows through the map
- reshape the metabolites that are excreted-> I think I will try this method

Feel free to add other suggestions.
Kind regards,
Julia

### Type of revision

- [ ] Major
- [ ] Minor
- [ ] Deletion

**Comment by egonw** [comment]
cc @DeniseSl22

**Comment by Chris-Evelo** [comment]
We have had pathways like this in the (NuGO) past, where we had metabolites excreted from organs to plasma. Apart from the best way to show this, there is also an analytical aspect to this kind of situation. Biomarkers are typically measured in another compartment (urine, plasma, exhaled air) than where they are formed. Sometimes you would want to be able to show that from the analytical results, like showing measured urinary metabolites only in urine and modeled metabolite formation only in the tissue. That suggests that you need different identifiers for the two, or an identifier with a specification. We solved that at the time by creating artificial identifiers (If I remember correctly something like XXX_P for plasma metabolites where XXX is the usual identifier). Of course that is artificial, although you could extend it to a general principle where you would have to parse the attribute from the identifier first. Maybe we should solve this?

**Reply by DeniseSl22** [reply]
@Chris-Evelo I wouldn't go into the artificial ID direction, since PV4 will allow to add the localization of a DataNode @mkutmon . @JuliaBatUM , for now I would indeed use the 'basic arrow', but that let this point to another metabolite datanode with the same ID. You could add in the comment where this protein is localizes, and add to the comment of the interaction arrow that this is a translocation.

**Reply by DeniseSl22** [reply]
I also agree with @khanspers comment, that the pathway is already quite large so adding more datanodes will clutter that even more. There's no easy fix for that unfortunately, but my comments above where more keeping the downstream data analysis in mind.

**Reply by Chris-Evelo** [reply]
At the time people came up with that artificial ID solution since they had data on two different locations and wanted to show that in the pathways. Of course, we "fixed" those pathways later, which had the unintended side effect that the analysis done at the time cannot be reproduced. But don't get me wrong. I do not advocate using such artificial IDs. I would rather see a more general qualifier option where we can indicate more information. That is not only about the location but can also be about processes that involve conformational changes specific forms of activation and modification and such. In such cases, we typically have gene products with the same ID on two sides of a connecting arrow and it would be good to be able to discriminate the two during analysis since people can be able to measure both forms.

**Comment by khanspers** [comment]
Hi @JuliaBatUM. A couple of comments:
`add a secretion reaction and have the metabolite drawn outside the cell -> I think this would make the map even more unclear as it would add more reactions and more arrows through the map`
=> Typically, we recommend this style of representing the secreted node twice in the two different compartments, but since this pathway is quite large, I agree that it doesn't make sense. The current representation of arrows from the relevant metabolites to a label or pathway node indicating "excreted in urine" is clear, and efficient in terms of space.

`reshape the metabolites that are excreted-> I think I will try this method` 
=> I'm not sure changing the shape of the node is necessary, as the information is already represented with the interaction to the label/node (above). But if you decide to go with a different node shape, make sure to add that information to a legend.
